.github/workflows/pr-build-test.yml

```
@@ -0,0 +1,14 @@
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - dual-verse-dag
+
+jobs:
+  build-and-test:
+    uses: ./.github/workflows/core-build-test.yml
+    with:
+      runner: self-hosted
+    secrets: inherit
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)